commit aabd2479cbaeb3f8151e41f6860ba24762753e6a
Author: Felix Yan <felixonmars@archlinux.org>
Date:   Tue May 15 15:50:26 2018 +0800

    fix: generate correct blur filename
    
    A space was added between the filename and path when calculating md5sum, this causes the generated blur files useless.
    
    Before:
    $ echo -n "/usr/share/wallpapers/deepin/ Hummingbird_by_Shu_Le.jpg" |
    md5sum
    c85efb9df57e04a9aa704f2c29f8517c  -
    
    Fixed:
    $ echo -n /usr/share/wallpapers/deepin/Hummingbird_by_Shu_Le.jpg |
    md5sum
    f1b59d03d44b2c3babaa7769b3f905b3  -
    
    Change-Id: Ie641236f02b0303f45b8769987553c12005e671c

diff --git a/Makefile b/Makefile
index 19f835a..57cb3b0 100644
--- a/Makefile
+++ b/Makefile
@@ -10,7 +10,7 @@ prepare:
 	@mkdir -p image-blur
 
 blur/%: 
-	/usr/lib/deepin-api/image-blur --sigma 30 $(subst blur/,,$@) image-blur/$(call md5sum, ${@F}).jpg
+	/usr/lib/deepin-api/image-blur --sigma 30 $(subst blur/,,$@) image-blur/$(call md5sum,${@F}).jpg
 
 clean:
 	rm -rf image-blur
